client/state/current-user/selectors.js

```diff
@@ -32,6 +32,15 @@ export function getCurrentUserLocale( state ) {
 
 	return user.localeSlug || null;
 }
+/**
+ * Returns the currency code for the current user.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {?String}        Current currency code
+ */
+export function getCurrentUserCurrencyCode( state ) {
+	return get( state.currentUser, 'currencyCode', null );
```

Comment:
The `get` here doesn't hurt but I suppose we wouldn't be calling a selector with a malformed state snapshot. Updated in 0278c4a